docs/release-notes/8.6.asciidoc

```diff
@@ -0,0 +1,99 @@
+[[release-notes-header-8.6.0]]
+== 8.6
+
+[discrete]
+[[release-notes-8.6.0]]
+=== 8.6.0
+
+[discrete]
+[[known-issue-8.6.0]]
+==== Known issues
+There are no known issues in 8.6.0.
+
+[discrete]
+[[breaking-changes-8.6.0]]
+==== Breaking changes
```

Comment:
Should this section have a "There are no known issues..." line, like the others above and below?

Reply:
Previously, we haven't done this and only doc known issues when requested -- i.e., when we're notified that a known bug needs to be release-noted. That said, if you think it'd be useful to include a known issues section, regardless of whether known issues exist, I'd be open to discussing it. :)

Reply:
We already have a "Known issues" section, with a statement in it; same with "Deprecations." The problem I was seeing was that we also have a "Breaking changes" section, but there's nothing at all in it, which looks like something's missing (esp compared to these other two sections).

<img width="638" alt="image" src="https://user-images.githubusercontent.com/87339667/210812156-d6a367e4-d520-411b-a833-3d51b0ef7760.png">

I think I slightly misspoke above -- I think the "Breaking changes" section should include the statement "There are no breaking changes in 8.6.0" in it, instead of just being an empty section heading with no content.

Reply:
I had that line in there previously, and removed it as Nastasha suggested. I don't have strong feelings either way, since I think having the section blank communicates that there are none, but I agree we should be consistent with the other sections above and below. Unless anyone objects I'll just remove the "There are no..." above and below, as well.

Reply:
I actually think having section headings without any content is not a great practice in general, especially if there would be three empty sections in a row, just three headings without content. Including the statements might be a little overkill (like the old "this page intentionally left blank"), but it doesn't leave any doubt that the sections are finished and complete.